irohad/main/config/flags.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_FLAGS_HPP_
+#define IROHA_FLAGS_HPP_
+
+#include <gflags/gflags.h>
+
+/**
+ * It is safe to include this file anywhere for flag usage. Don't forget to link
+ * `irohad-flags` to your target.
+ *
+ * However, it is not recommended to use flags. Use iroha::config::Config.
+ */
+
+/// -new_ledger
+DECLARE_bool(create);
+DECLARE_string(genesis_block);
+
+/// Redis
+// redis host
```

Comment:
remove this meaningless comments plz :( (section are fine)

Reply:
They are not meaningless. With these  comments it is easy to find the code related to specific flag.

Reply:
```
//redis host
declare redis_host
```
orly? if you are able to find by the comment then you'll be able to find by the var